QuestionBot/QuestionBot.UnitTests/Model/QuestionMessageListenerTests.cs

```diff
@@ -0,0 +1,123 @@
+﻿using System;
+using System.Linq;
+using NUnit.Framework;
+using System.Collections.Generic;
+using QuestionBot.Model;
+
+namespace QuestionBot.UnitTests.Model{
+
+    [TestFixture]
+    class QuestionMessageListenerTests{
+        private IMessageListener _testListener;
+        private IStore _storeTest;
+
+        [SetUp]
+        public void Setup(){
+            _storeTest = new InMemoryStore();
+            _testListener = new QuestionMessageListener(_storeTest);
```

Comment:
I realized that I forgot to go over `Moq` in our last lesson. Let's put a hold on these tests until we meet again.

The problem here is that this file tests `QuestionMessageListener`. But what if there is a bug in `InMemoryStore`? Then these tests will fail (and presumably `InMemoryStoreTests`) making it hard to see where the real problem is. We want to insulate this file from failures in dependencies (dependencies being `InMemoryStore` in this case). `Moq` is a mocking library that allows us to do this. We'll go over it next Monday.